packages/expo-location/android/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
   <application>
     <service
       android:name=".services.LocationTaskService"
-      android:exported="false" />
+      android:exported="false"
+      android:foregroundServiceType="location" />
```

Comment:
Android automatically uses the "manifest service type" when starting the foreground service. It's possible to add more, and start the service for specific types. See [this documentation](https://developer.android.com/guide/components/foreground-services#loc-camera)